lambda/qnabot-common-layer/Makefile

```diff
@@ -0,0 +1,16 @@
+NAME=$(shell basename $(shell pwd))
+DST=../../build/lambda/$(NAME).zip
+
+RESOURCES := $(shell  find . | grep -v node_modules | grep -v test )
+
+$(DST): $(RESOURCES) 
+	echo "Building $(NAME)"; 
+	npm install  -production
+	rm -r ./nodejs || true
+	mkdir ./nodejs
+	mv node_modules ./nodejs/node_modules || true
+	mkdir ./nodejs/node_modules || true  #if no node_modules folder exists because there was nothing to copy, create it
+	mkdir ./nodejs/node_modules/qnabot
+	cp -R qnabot ./nodejs/node_modules/
+	zip -FSr $(DST) .
```

Comment:
It would be more direct to use `zip -FSr $DST nodejs` instead. Therefore, removing the need `zip -d $(DST) qnabot` in the line. Also while not impacting anything, the Makefile, and package*.json (or any other files at this directory level) would not be placed in the zip.